os/arch/arm/src/s5j/s5j_i2s.c

```diff
@@ -155,9 +155,12 @@
 #undef CONFIG_I2S_TXS_DMACH
 #undef CONFIG_I2S_RX_DMACH
 
-#define CONFIG_I2S_TXP_DMACH    0
-#define CONFIG_I2S_TXS_DMACH    1
-#define CONFIG_I2S_RX_DMACH     2
+#define CONFIG_I2S_TXP_DMACH	0
+#define CONFIG_I2S_TXS_DMACH	1
+#define CONFIG_I2S_RX_DMACH	2
+#define CONFIG_I2S_TXP_DMACH	0
```

Comment:
You removed my comments. 158~160 and 161~163 dupplicated

Reply:
done